node/full_client.go

```diff
@@ -496,7 +496,15 @@ func (c *FullClient) Commit(ctx context.Context, height *int64) (*ctypes.ResultC
 	if err != nil {
 		return nil, err
 	}
-	commit := com.ToABCICommit(heightValue, b.Hash())
+
+	// we only have one centralize sequencer = 1 validators
```

Comment:
@Manav-Aggarwal we need update valset in header header because in line 723, `m.getCommit`, validator will sign the entire header. If we remove this update here, it will break the sign and verify header process.

Reply:
I think we can remove the valset update outside else block instead of remove this.

Reply:
That's fine by me too :)